app/controllers/v6/assessments_controller.rb

```diff
@@ -27,33 +27,31 @@ def create
                                 applicant_outgoings)
 
         partner_params = full_assessment_params[:partner]
-        if partner_params.present?
-          partner_dependants = dependants partner_params, create.assessment.submission_date
-          render_unprocessable(dependant_errors(partner_dependants)) && return if partner_dependants.reject(&:valid?).any?
+        calculation_output = if partner_params.present?
```

Comment:
@starswan I thought of removing else by  assigning partner to nil before condition. What do you think?